ngraph/core/include/ngraph/check.hpp

```diff
@@ -18,11 +18,11 @@ using ov::CheckFailure;
 using ov::CheckLocInfo;
 }  // namespace ngraph
 
-#define NGRAPH_CHECK_HELPER2(exc_class, ctx, check, ...) OV_CHECK_HELPER2(exc_class, ctx, check, __VA_ARGS__)
+#define NGRAPH_CHECK_HELPER2(exc_class, ctx, check, ...) OPENVINO_CHECK_HELPER2(exc_class, ctx, check, __VA_ARGS__)
 
-#define NGRAPH_CHECK_HELPER1(exc_class, ctx, check) OV_CHECK_HELPER1(exc_class, ctx, check)
+#define NGRAPH_CHECK_HELPER1(exc_class, ctx, check) OPENVINO_CHECK_HELPER1(exc_class, ctx, check)
 
-#define NGRAPH_CHECK(...) OV_CHECK(__VA_ARGS__)
+#define NGRAPH_CHECK(...) OPENVINO_CHECK(__VA_ARGS__)
```

Comment:
do we need a dedicated header with check.hpp? can we put this stuff to core/except.hpp?

Reply:
Fixed